kubernetes/kube-state-metrics-cluster-role.yaml

```diff
@@ -15,8 +18,11 @@ rules:
   verbs: ["list", "watch"]
 - apiGroups: ["extensions"]
   resources:
-  - daemonsets
   - deployments
+  verbs: ["list", "watch", "get", "update"]
```

Comment:
Letting this update any replicaset and daemonset in the system doesn't seem appropriate. I'd expect those permissions limited to the namespace and specific replicaset/daemonset it is responsible for. Also, the update permission belongs in a separate role specific to the nanny, since someone who is not using the nanny should not need to give write permission to their metrics pod